_pages/Long-term-support/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+permalink: /Long-term-support/
+---
+
+## Long term engineering support
```

Comment:
The layout doesn't look so great now. Header with ## is automatically made yellow (see other pages).

Reply:
<img width="1069" alt="Screenshot 2022-02-03 at 16 42 05" src="https://user-images.githubusercontent.com/3865773/152379303-855dea9b-191d-4f76-b25d-6632b89dd509.png">

Reply:
modified layout